spec/features/response_condition_spec.rb

```diff
@@ -4,7 +4,21 @@
   before do
     @user = create(:user)
     @form = create(:form, name: 'Foo',
-      question_types: %w(long_text text integer text decimal select_one multilevel_select_one multilevel_select_one select_multiple datetime date time text))
+      question_types: [
+        "long_text",             #  0
```

Comment:
These little hints are great, eventually I'd like to try doing this with maybe a hash (using `.keys` to get the array for the form builder), or try to find some other way to make these sorts of specs a little more human-readable but the comment hints are a really great start.